@kiva/kv-components/vue/KvCarouselSlide.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
Is this component necessary anymore?

Reply:
I don't feel strongly about it, I can certainly remove it